docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -187,6 +194,8 @@ ALTER TABLE table_identifier [ partition_spec ] SET SERDE serde_class_name
 `ALTER TABLE SET` command can also be used for changing the file location and file format for 
 existing tables. 
 
+If the table is cached, the `ALTER TABLE .. SET LOCATION` command clears cached data of the table and all its dependents that refer to it. The cache will be lazily filled when the next time the table or the dependents are accessed.
```

Comment:
We should probably put this in the beginning only once:
```
### Description

`ALTER TABLE` statement changes the schema or properties of a table. If the table is cached ...
```

Reply:
- It is not applicable to all commands
- I modified not only `ALTER TABLE`